bin/requirements.yml

```diff
@@ -9,5 +8,5 @@
 - src: jdauphant.nginx
   version: v2.7.4
 
-- src: geerlingguy.certbot
-  version: 1.0.1
+- src: coopdevs.certbot_nginx
```

Comment:
where can I see this role?

Reply:
Sorry, added https://github.com/coopdevs/certbot_nginx to description